docs/node/quickstart.mdx

```diff
@@ -33,6 +43,9 @@ To generate the configs for 3 validators in `./kwil-testnet`, run:
 kwil-admin setup testnet -v 3 --hostnames "localhost,localhost,localhost" --output-dir ./kwil-testnet
 ```
 
+This creates three nodes that can run on the same host for evaluation.
+A production network would comprise nodes on different machines, but with the same `genesis.json` file.
+
 ### Run Nodes
 
 To run the nodes, we can use the `kwild` binary in 3 separate terminals.  We will need to specify the `--root_dir` flag to
```

Comment:
Same for above, should we add quickstart commands for starting postgres for these three nodes? Might be annoying with all instances and nodes running on the same machine 😅

Reply:
Yup, this one is tricky.  Some related Qs in https://github.com/kwilteam/kwil-db/pull/557 description.
The benefit of the multi-node quick start is somewhat lost on me, but if we want to keep it, it'll have three different `docker run` commands with different ports and data directory mounts specified, plus `kwild` commands modified with `--app.pg_db_port` modified to match.